### PR TITLE
UI | landing page - chatbot page

### LINKED ISSUE #3: Create a UI to start interacting with the model

### PR DESCRIPTION
**Pull request by nouralmulhem**
this PR include the first version of the chatbot UI including both the landing page and chatting page

creating UI for the chatbot issue: [link](https://github.com/nouralmulhem/Enhancing-LLM-with-Jenkins-Knowledge/issues/3)

### CODE REVIEW

File: FE/src/index.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+/* Handle */
+::-webkit-scrollbar-thumb {
+  background: #f50057; 
+  border-radius: 10px;
+}
+
+/* Handle on hover */
+::-webkit-scrollbar-thumb:hover {
+  background: #f50057; 
+}
```
Comment (krisstern) [inline FE/src/index.css:41]:
```suggestion
}

```

**Review by krisstern — APPROVED** [review]
I think this PR looks good. It is very well organised and includes all the essential config. Good job!

File: FE/src/index.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
```
Comment (krisstern) [inline FE/src/index.css:3]: For the font-family, I think we will eventually have to align with the practices currently adopted at [https://github.com/jenkins-infra/jenkins.io](https://github.com/jenkins-infra/jenkins.io). But we can do that in a follow-up PR.

**Review by harsh-ps-2003 — APPROVED** [review]
LGTM !
Can we have a loading animation or something to tell that the model is going to give a response and is not stuck or facing an error in the later iterations of the UI? @nouralmulhem

File: FE/src/layouts/LandingBody/LandingBody.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        can help users with their queries.
+      </Typography>
+
+      <ButtonsWrapper>
+        <Button
+          color="primary"
+          variant="contained"
+          size="large"
+          component={Link}
+          to="/dashboard"
```
Comment (shivaylamba) [inline FE/src/layouts/LandingBody/LandingBody.tsx:28]: The buttons should also have aria-labels for better accessbilility.

**Review by shivaylamba — APPROVED** [review]

**Pull request merged**